react/lib/components/Widget/WidgetContainer.tsx

```diff
@@ -187,6 +190,13 @@ export const WidgetContainer: React.FunctionComponent<WidgetContainerProps> =
             onSuccess?.(transaction);
           } else {
             onTransaction?.(transaction);
+            if (transactionText){
+              setNewTxText(transactionText)
+              setTimeout(() => {
+                setNewTxText('')
+              }, 10000)
+            }
```

Comment:
I wasn't really expecting this... I think it's better to stick with the standard toast and perhaps we can make a new task to better differentiate them so people don't mistakenly think they paid the full correct amount when they did not.

Reply:
On second thought - @chedieck thoughts on this? Maybe John also has some ideas how best to visually communicate the custom message while also making it clear that less than the full amount was paid.

Reply:
Maybe call it "Unexpected payment" or something like that by default? Probably some other more precise adjective would be better.

As for the PR, this `10000` should be a constant instead of a hardcoded number.